### DISCUSSION TITLE
javascript PTC website script

### DISCUSSION THREAD

**Discussion by fayfay65** [symfony/symfony#46579] [Q&A] [answered]
Hello everybody,
Recently there has been a flurry of "order grabbing" apps—Rakutenshop, Shopkick, Topshop 360, —which pay commission for grabbing orders from shopping websites such as Amazon, Wish, Shopee, etc (examples: "m.joomqq.com" , "dmm444.com" and too many more, but although there are too many to count, they all share the very same script with different front end templates.
i've been trying to find the source (maybe seller) of that script if anyone has an idea or could build a better one with the same concept for a good price.
you can email me at: ourhou.faissal87@gmail.com
thanks

**Comment by lyrixx** [comment, marked as answer]
This has nothing to do with symfony... I mark it as answered to clean the discussion space.
